libbeat/outputs/otelconsumer/otelconsumer_test.go

```diff
@@ -186,20 +186,19 @@ func TestPublish(t *testing.T) {
 		batch := outest.NewBatch(event3)
 		batch.Events()[0].Content.Timestamp = time.Date(2025, time.January, 29, 9, 2, 39, 0, time.UTC)
 
-		var timestamp string
+		var bodytimestamp string
 		otelConsumer := makeOtelConsumer(t, func(ctx context.Context, ld plog.Logs) error {
 			record := ld.ResourceLogs().At(0).ScopeLogs().At(0).LogRecords().At(0)
```

Comment:
Let's make sure we assert that the timestamp of the LogRecord matches the body timestamp as well, `record.Timestamp` in this case.